cmd/minikube/cmd/update-context.go

```diff
@@ -33,14 +33,17 @@ var updateContextCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cname := ClusterFlagValue()
 		co := mustload.Running(cname)
-		updated, err := kubeconfig.UpdateIP(co.DriverIP, cname, kubeconfig.PathFromEnv())
+		ip := co.CP.ForwardedIP
+
+		// ??? For KIC, should we also update the port ???
```

Comment:
I believe update context happens only on a running or stopped container.

As long as the container doesn't get recreated we don't need to update the port. 

But I could see this be a problem in future.
Like if we re-create a failing kic node and the call update context.

So it would be a good thing if we get the forwarded port value for the user configured Api server port or the default Api server port. Using oci.ForwardedPort.

Reply:
Will resolve in a follow-up PR, as the delta for handling this properly is fairly large.